### PR TITLE
Header icons

### PR DESCRIPTION
**Pull request by jivey**

### CODE REVIEW

File: src/components/ExerciseIcons.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+`;
+
+interface PopoverItemProps {
+  text: string;
+  children: JSX.Element;
+  wrapperProps?: {
+    as: keyof JSX.IntrinsicElements;
+    href?: string;
+    target?: string;
+  }
```
Comment (jivey) [inline src/components/ExerciseIcons.tsx:44]: I don't love typing it like this but I failed at figuring out the right generic and gave up 😬

File: src/components/Icons.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import HTMLComment from 'react-html-comment';
```
Comment (jivey) [inline src/components/Icons.tsx:1]: TIL React doesn't support html comments but the license requires the attribution to be embedded.

**Comment by jivey** [comment]
There are a couple of follow up items we may need to do later but I want to try and slice off this work for now... 

1. The width of the popover should be flexible so it can be narrow or widen based on the text content.
2. We may need to support more types than multiple-choice and two-step? I didn't bring over the other type text from Tutor because I think they will remain specific to Tutor stuff (personalized, spaced practice)

File: src/components/Icons.tsx (hunk 2 above)
Comment (bethshook) [reply src/components/Icons.tsx:1]: hmm I've always used the fontawesome react package - does that not include some alternative?

Comment (jivey) [reply src/components/Icons.tsx:1]: I haven't used that, maybe it does something similar... I was worried about bundle size with that but I see they say importing the exact icons you need will let tree shaking do it's thing, so maybe that's a better alternative than this one-off package. I'll see what the build size is with the font awesome package when I get a chance.

Comment (jivey) [reply src/components/Icons.tsx:1]: Or yeah from our convo if you want to pull font awesome in instead that would be great!

**Review by bethshook — APPROVED** [review]

**Pull request merged**